Lesson3/Shopper/Shopper/Model/ShoppingModel.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+class Shopping: ShoppingList {
+    var shoppingList: [String] = []
+    
+    func add(_ item: String) -> Bool {
+        if shoppingList.count == 10 {
```

Comment:
What do you think, how it's possible to bypass the following check
```swift
shoppingList.count == 10
```
?

Reply:
it is still possible to change the shoppingList from outside. I must change the access level. Corrected it